eslint/babel-eslint-parser/package.json

```diff
@@ -31,7 +31,6 @@
     "@babel/core": "^7.2.0",
     "@babel/eslint-shared-fixtures": "*",
     "eslint": "^6.0.1",
-    "espree": "^6.0.0",
```

Comment:
Thoughts on this? Seems like we should ensure we're using the version of Espree that's included with the version of ESLint we're testing against.

Reply:
i think it should be a combo peer dep and dep, that way `npm ls` will fail (and `npm install` in npm 7+) when the peer dep isn't met.

Reply:
@nicolo-ribaudo @JLHwung @existentialism Thoughts?

Reply:
Now that we use `require.resolve` to get ESLint's dep, this shouldn't be needed anymore.